index.js

```diff
@@ -12,11 +11,20 @@ import { rawToHex, rawHashToHex, reverseHash, str2hex, hex2b64, str2utf8hex } fr
 import { sprintf } from "sprintf-js";
 import { globalCryptoShim } from "./helpers/random";
 
+const session = require('trezor-connect').default;
+const { TRANSPORT_EVENT, UI, UI_EVENT, DEVICE_EVENT, DEVICE, CONNECT } = require('trezor-connect');
+
 // app constants
+const CHANGE = 'device-changed'
+const DISCONNECT = 'device-disconnect'
+const AQUIRED = 'acquired'
+const NOBACKUP = 'no-backup'
 const coin = "Decred Testnet";
 const coinNetwork = networks.decred;
-const walletCredentials = WalletCredentials("127.0.0.1", 19121,
-    "/home/user/.config/decrediton/wallets/testnet/trezor/rpc.cert");
+const walletCredentials = WalletCredentials("127.0.0.1", 19576,
+    "/home/joe/dcrtesting/rpc.cert");
+// const walletCredentials = WalletCredentials("127.0.0.1", 19121,
+    // "/home/user/.config/decrediton/wallets/testnet/trezor/rpc.cert");
```

Comment:
Revert these back.

Ideally this should be something like (untested):

```js
let walletCredentials;
try {
   walletCredentials = require("./walletcreds.js");
} catch (err) {
  const exampleCreds = 'module.exports = {address: "127.0.0.1", port: 19212, certificate: "/home/user/.config/decrediton/wallets/testnet/trezor/rpc.cert"}'
  console.log("Create a local walletcreds.js file with the following content\n:", exampleCreds);
  return
}
```

So that we can easily switch the creds.

Reply:
Also needs to be made to work with grpc client certs something or other...